internal/shim-sgx/src/handler.rs

```diff
@@ -145,6 +146,23 @@ impl<'a> SyscallHandler for Handler<'a> {
         self.exit(1)
     }
 
+    #[inline]
```

Comment:
Why inline? Just curious.

Reply:
just a hint that the compiler is allowed to inline